modules/assim.sequential/R/sda.enkf.R

```diff
@@ -8,13 +8,14 @@
 ##' @param IC          initial conditions
 ##' @param Q           process covariance matrix given if there is no data to estimate it
 ##' @param adjustment  flag for using ensemble adjustment filter or not
+##' @param restart
 ##' 
 ##' @description State Variable Data Assimilation: Ensemble Kalman Filter
 ##' 
 ##' @return NONE
 ##' @export
 ##' 
-sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustment = TRUE) {
+sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustment = TRUE, restart=NULL) {
```

Comment:
Why is the restart param added? Maybe add description above?

Reply:
@mdietze do you want something specific?

Reply:
Yeah, I put a lot of that detail in the PR I sent to @istfer but you are right that more documentation should go directly into the code. Istem, I'll write that up and send it to you so you can insert it into the PR (probably easier than me doing another PR to you).

In a nutshell, restart is to allow forecasts to be iterative. e.g you ran a forecast from 2017-2027, it's now 2018 and you have new data, you want to restart from what you previously forecasted would happen in 2018. The code starts a completely new workflow (ID, folders, etc) but pulls over the previous forecast for that timestep, previous parameters, previous pointers to which met ensemble member you were using, etc. so that you restart the ensemble from the right place. The code then skips the first Forecast and dives directly into the first Analysis